crates/vm/vm.rs

```diff
@@ -153,6 +157,62 @@ cfg_if::cfg_if! {
 
         }
 
+        /// Calls the EIP-2935 process block hashes history system call contract
```

Comment:
There are two possibilities. 

1. Ideally, we can abstract the logic between levm/revm and implement `process_block_hash_history`, agnostic of the implementation of the vm (levm or revm).
2. If 1 is impossible, we should at least have a `levm` and `revm` folder or module were we specify these implementations.